llumnix/backends/vllm/worker.py

```diff
@@ -66,135 +84,63 @@ def init_migration(self, num_migration_cache_blocks: int, src_worker_handle_list
                 node_id=ray.get_runtime_context().get_node_id(),
```

Comment:
The node_id should not get through ray's runtime context after support init llumlet in manager, you can reference [https://github.com/AlibabaPAI/llumnix/pull/11](url) to revise it. The reason is that when we init llumlet in manager, the runtime context of llumlet will be manager process.

Reply:
Resolved in https://github.com/AlibabaPAI/llumnix/pull/11.